### DISCUSSION TITLE
Install on TrueNAS Scale Docker not working...

### DISCUSSION THREAD

**Discussion by xcalybur** [TandoorRecipes/recipes#1865] [Q&A]
Hello all,

I m trying to use the TrueCharts docker image for Tandoor/Recipes. I'm using all of the default options and it take forever to deploy (Like 2-4 hours). Afterwards I can create the superuser account in the web interface, but when I try to create a recipe or import from any website it gives me the error, "Failure: There was an error creating a resource!"

I think there is something wrong with the install, but I don't know what options to change. Has anyone done this successfully and how did you do it?

Thank you so much.

**Comment by gysmo8469** [comment]
The most recent install from truecharts works out of box.

**Comment by lukaszervudakis** [comment]
I got the same issue with the most recent version from truecharts. Any idea?

**Reply by gysmo8469** [reply]
Sorry unfortunately I no longer have truenas scale to test this installation.

**Comment by gysmo8469** [comment]
> I got the same issue with the most recent version from truecharts. Any idea?

What I could suggest is maybe look at folder permissions. When I used scale most times it would be permissions. See if the volumes are writable from within the application.
